node/pkg/watchers/evm/ccq_backfill.go

```diff
@@ -0,0 +1,303 @@
+package evm
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/certusone/wormhole/node/pkg/watchers/evm/connectors"
+
+	"go.uber.org/zap"
+
+	ethHexUtil "github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rpc"
+	ethRpc "github.com/ethereum/go-ethereum/rpc"
+)
+
+const CCQ_MAX_BATCH_SIZE = int64(1000)
+const CCQ_TIMESTAMP_RANGE_IN_MS = uint64(30 * 60)
+
+type ccqBackfillRequest struct {
+	timestamp uint64
+}
+
+// ccqRequestBackfill submits a request to backfill a gap in the timestamp cache.
```

Comment:
Could you add to this comment saying the timestamp is in microseconds?

Reply:
Actually it's in seconds. I added a comment.